zio-schema/shared/src/main/scala/zio/schema/Diff.scala

```diff
@@ -315,6 +316,8 @@ object Differ {
     case Schema.Enum21(c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, c16, c17, c18, c19, c20, c21, _)      => enumN(c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, c16, c17, c18, c19, c20, c21)
     case Schema.Enum22(c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, c16, c17, c18, c19, c20, c21, c22, _) => enumN(c1, c2, c3, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, c16, c17, c18, c19, c20, c21, c22)
     case Schema.EnumN(cs, _)                                                                                                   => enumN(cs.toSeq: _*)
+    case Schema.Dynamic(_)                                                                                                     => ??? // TODO
+    case Schema.SemiDynamic(_, _)                                                                                              => ??? // TODO
```

Comment:
Not quite sure how to handle diffs on `Dynamic` but for `SemiDynamic` I think it should be relatively straightforward. It would just be a diff on the values (using the packaged schema).

Reply:
Sure I just wanted to have working `SemiDynamic` serialization asap so I can continue with zio-flow and defer this to a later PR if possible.